### PR TITLE
added com.github.rajsolai.textsnatcher manifests

### PR DESCRIPTION
**Pull request by RajSolai**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### Additional Permissions
- '--filesystem=/home' : *Used to save screenshots on User directory*
- '--talk-name=org.gnome.Shell.Screenshot' : *Used to talk to gnome screenshot*

### CODE REVIEW

**Comment by hfiguiere** [comment]
PR is suppose to be against the `new-pr` branch.

File: com.github.rajsolai.textsnatcher.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+app-id: com.github.rajsolai.textsnatcher
+
+runtime: org.gnome.Platform
+runtime-version: "40"
```
Comment (hfiguiere) [inline com.github.rajsolai.textsnatcher.yml:4]:
```suggestion
runtime-version: "41"
```

File: com.github.rajsolai.textsnatcher.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+base: io.elementary.BaseApp
+base-version: "juno-20.08"
```
Comment (hfiguiere) [inline com.github.rajsolai.textsnatcher.yml:6]:
```suggestion
base-version: "juno-21.08"
```

File: com.github.rajsolai.textsnatcher.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.textsnatcher
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=/home"
```
Comment (hfiguiere) [inline com.github.rajsolai.textsnatcher.yml:16]:
```suggestion
  - "--filesystem=/home"
```

1. this won't work
2. you should set it to `xdg-pictures`

File: com.github.rajsolai.textsnatcher.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  - "--talk-name=org.gnome.Shell.Screenshot"
+  - "--system-talk-name=org.freedesktop.Accounts"
```
Comment (hfiguiere) [inline com.github.rajsolai.textsnatcher.yml:18]: why is this needed?

File: com.github.rajsolai.textsnatcher.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+modules:
+  - name: com.github.rajsolai.textsnatcher
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/RajSolai/TextSnatcher/archive/refs/tags/1.0.0.tar.gz
+        sha256: 5858702455a646b35a8496a44efab7216b780cdc9a0bc89e8337991c1ac1e0f5
+  - tesseract.yml
```
Comment (hfiguiere) [inline com.github.rajsolai.textsnatcher.yml:27]: splitting this out is uneccessary and make the maintainance harder.

**Comment by hfiguiere** [comment]
bot, build com.github.rajsolai.textsnatcher

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
appdata fail to validate:
```
• attribute-invalid     : <release> timestamp is in the future
```

File: com.github.rajsolai.textsnatcher.yml (hunk 1 above)
Comment (RajSolai) [reply com.github.rajsolai.textsnatcher.yml:4]: Sure I will update the runtime version :+1:

File: com.github.rajsolai.textsnatcher.yml (hunk 2 above)
Comment (RajSolai) [reply com.github.rajsolai.textsnatcher.yml:6]: I didn't know of a newer base app version thanks , I will update :+1:

File: com.github.rajsolai.textsnatcher.yml (hunk 3 above)
Comment (RajSolai) [reply com.github.rajsolai.textsnatcher.yml:16]: Sure I will set it to pictures directory :+1:

File: com.github.rajsolai.textsnatcher.yml (hunk 4 above)
Comment (RajSolai) [reply com.github.rajsolai.textsnatcher.yml:18]: [--system-talk-name=org.freedesktop.Accounts](https://valadoc.org/granite/Granite.Settings.prefers_color_scheme.html) Granite Needs this permission for prefers_colors_scheme method.

"--talk-name=org.gnome.Shell.Screenshot" is used to access screenshot dbus-service

File: com.github.rajsolai.textsnatcher.yml (hunk 5 above)
Comment (RajSolai) [reply com.github.rajsolai.textsnatcher.yml:27]: Sure I will make the manifest as one :+1:

**Comment by RajSolai** [comment]
> appdata fail to validate:
> 
> ```
> • attribute-invalid     : <release> timestamp is in the future
> ```

Oops, I will change it

**Comment by RajSolai** [comment]
I am closing this PR, and Open a new PR against the new-pr branch, Thanks for the Review :smile:
